ruleset/testing/tests/cisco_ftd.ini

```diff
@@ -1,198 +1,176 @@
-[Severity 1: alerts]
+;   Copyright (C) 2015-2021, Wazuh Inc.
+;
+;   Tests for products:
+;     Cisco FTD
+;
+;   Sample logs source: https://www.cisco.com/c/en/us/td/docs/security/firepower/Syslogs/b_fptd_syslog_guide.html
+;
+
+[Cisco FTD: High severity alert]
 log 1 pass = %FTD-1-101001: (Primary) Failover cable OK.
 log 2 pass = %FTD-1-101002: (Primary) Bad failover cable.
-
 rule = 91501
 alert = 7
 decoder = cisco-ftd
 
-[91520]
-log 22 pass = %FTD-1-105005: (Primary) Lost Failover communications with mate on interface interface_name.
-log 26 pass = %FTD-1-105009: (Primary) Testing on interface interface_name {Passed|Failed}.
-log 41 pass = %FTD-1-105043: (Primary) Failover interface failed
-
+[Cisco FTD: Firewall failover pair communication problem]
+log 1 pass = %FTD-1-105005: (Primary) Lost Failover communications with mate on interface interface_name.
+log 2 pass = %FTD-1-105009: (Primary) Testing on interface interface_name {Passed|Failed}.
+log 3 pass = %FTD-1-105043: (Primary) Failover interface failed
 rule = 91520
```

Comment:
Tests are in random order. Could we order the tests according to the rule ID as the rest of the test files?